projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/delivery-mode/delivery-mode.component.html

```diff
@@ -22,30 +20,24 @@ <h3 class="cx-delivery-mode-form__title d-none d-lg-block d-xl-block">
             formControlName="deliveryModeId"
           />
           <label
-            class="cx-delivery-mode-form__label form-check-label
+            class="cx-delivery-label form-check-label
             form-radio-label"
             for="deliveryMode-{{ mode.code }}"
           >
-            <div class="cx-delivery-mode-form__label--shipping-mode">
-              {{ mode.name }}
-            </div>
-            <div class="cx-delivery-mode-form__label--price">
+            <div class="cx-delivery-shipping">{{ mode.name }}</div>
+            <div class="cx-delivery-price">
               {{ mode.deliveryCost.formattedValue }}
             </div>
-            <div class="cx-delivery-mode-form__label--details">
-              {{ mode.description }}
-            </div>
+            <div class="cx-delivery-details">{{ mode.description }}</div>
           </label>
         </div>
-
-        <!-- BUTTON SECTION -->
-        <div class="cx-delivery-mode-form__btns row">
-          <div class="col-sm-12 col-md-6">
+        <div class="row">
```

Comment:
You should add the class `cx-btns`. In the PR for #630 @developpeurweb changed `&__btns` to `cx-btns` in `checkout-media-style()`.